### PR TITLE
Development

### PR DESCRIPTION
**Pull request by emmanuelkamala**

### CODE REVIEW

**Comment by Torres-ssf** [comment]
### Great job 👍 
Nice work on this project. You got almost everything right. It just needs a few changes to be in accord with Microverse guidelines.
- [ ] All of the 8 sections (Top Story, More Stories, Featured Stories, Opinion, Latest News, In the Magazine, Editor's picks, Featured Slideshow) have to be a separated `section` tag each.

- [ ] The top story must be an article tag(inside the section tag).

- [ ] I think you placed the Latest News section inside the opinion section.

Kindly submit for code review again after these changes. Happy coding.

File: index.html [hunk 1]
```diff
@@ -0,0 +1,1004 @@
...
+                                    <img src="images/4-d-day.jpg" alt="D-Day Photos">
+                                    <div class="type">7</div>
+                                </div>
+                                <h2><a href="#">D-Day: Photos from the Invasion of Normandy</a></h2>
+                    
+                            </article>
+        </div>
+    </section>
+    <div class="more">
+        <div class="more-section u-s">
```
Comment (maelfosso) [inline index.html:799]: Use a `<section>` tag, please

File: index.html [hunk 2]
```diff
@@ -0,0 +1,1004 @@
...
+                                    </div>
+                                    <div class="more-text">
+                                        <h3 class="small-more-title"><a href="#">Senator Bernie Sanders Throws Support Behind Boston Graduate Students</a></h3>
+                                    </div>
+                                </div>
+                        
+                </div>
+    </div>
+</div>
+<div class="more-section more-world">
```
Comment (maelfosso) [inline index.html:859]: use `<section>` tag, please.

**Comment by maelfosso** [comment]
Hello,

You did a very good job. You improved your code. But there are some changes that you should do before submitting it.

- [ ] Use `<section>` in all the `<div.more..>`, please. I started indicating the first one.

- [ ] Also use `<section>` and `<article>` for `<div.subscribe-newsletter...>...</div>`, please. That block is not different from the others ones above.

Please take a look at all the comments above. Kindly, Make those changes and fill a new code review request.

**Comment by Torres-ssf** [comment]
### Great work @emmanuelkamala 👏
You can merge to master now. Don't forget to submit the form. Good luck with the next project.

**Comment by emmanuelkamala** [comment]
@Torres-ssf @maelfosso Thank you guys!

**Pull request merged**